packages/live-preview-sdk/src/inspectorMode/index.ts

```diff
@@ -46,9 +46,39 @@ export class InspectorMode {
       if (isInspectorActive) {
         this.sendAllElements();
       }
+    } else if (
+      data.method === LivePreviewPostMessageMethods.ENTRY_UPDATED ||
+      data.method === LivePreviewPostMessageMethods.ENTRY_SAVED
+    ) {
+      // for entry updates we need to wait a bit to make sure the DOM is updated
+      // then send the tagged elements again so that outlines are correct
+      this.handleResizing(250);
```

Comment:
Did you try adjusting the mutationObserver?
Maybe there would be something to optimize then we would not need this arbitrary number :)

Reply:
It wasn't working for me before but since passing `characterData: true` it works